email/blocker.go

```diff
@@ -85,83 +83,92 @@ func (b *Blocker) threadedBlockMessages() {
 	abuseDB := b.staticDatabase
 	logger := b.staticLogger
 
+	// create a new ticker
 	ticker := time.NewTicker(blockFrequency)
-	first := true
 
 	// start the loop
 	for {
-		// sleep until next iteration, sleeping at the start of the for loop
-		// allows to 'continue' on error.
-		if !first {
-			select {
-			case <-b.staticContext.Done():
-				logger.Debugln("Blocker context done")
+		logger.Debugln("Triggered")
```

Comment:
I'd make this a bit more descriptive. Someone unfamiliar with the codebase can only guess what "triggered" is referring to.

Reply:
The loggers have context `logger.WithField("module", "Blocker")` so the output reads `| time="2022-02-08 22:50:08" level=debug msg=Triggered module=Parser`. I made it more descriptive though and updated it to `logger.Debugln("threadedBlockMessages loop iteration triggered")`. I had a hard time coming up with something better 😅